src/ServicePulse.Host/vue/src/components/monitoring/EndpointListRow.vue

```diff
@@ -144,7 +139,7 @@ function formatGraphDecimal(input, deci) {
       <div class="no-side-padding">
         <SmallGraph :type="'critical-time'" :isdurationgraph="true" :plotdata="endpoint.metrics.criticalTime" :minimumyaxis="smallGraphsMinimumYAxis.criticalTime" :avglabelcolor="'#2700CB'" />
       </div>
-      <div class="no-side-padding sparkline-value" :class="{ negative: formatGraphDuration(endpoint.metrics.criticalTime).value < 0 }">
+      <div class="no-side-padding sparkline-value" :class="{ negative: parseFloat(formatGraphDuration(endpoint.metrics.criticalTime).value) < 0 }">
```

Comment:
should this be computed?

Reply:
i don't see any need to?

Reply:
I don't know if there are any benefits in this case; maybe the question is, should `formatGraphDuration` be called on every redraw?

Reply:
oh i see what you were referring to now: not just this line, but this div, calls formatGraphDuration 3 times for the same value. Will extract